docs/README.md

```diff
@@ -168,6 +144,102 @@ That's all you need to do. The `versions` directory is listed on server start to
 Because the navbar is automatically generated from the directory structure, the default ordering of the links under each section is alphabetical. However, for many sections, this is not ideal UX.
 So, if you wish to override the alphabetical ordering, manipulate page titles in **constants/navigation.js**.
 
+### Updating API reference docs
+
+The API reference docs are generated from the TypeScript source code.
+
+This section walks through the process of updating documentation for an Expo package. Throughout this document, we will assume we want to update TypeDoc definitions of property inside `expo-constants` as an example.
+
+#### Prerequisites
+
+Before proceeding, make sure you:
+
+- have [**expo/**](https://github.com/expo/expo) repo cloned on your machine
+  - make sure to [install `direnv`](https://direnv.net/docs/installation.html) and run `direnv allow` at the root of the **expo/** repo.
+- have gone through the steps mentioned in [**"Download and Setup" in the contribution guideline**](https://github.com/expo/expo/blob/main/CONTRIBUTING.md#-download-and-setup).
+- can run **expo/docs** repo **[locally](https://github.com/expo/expo/tree/main/docs#running-locally)**.
```

Comment:
```suggestion
- can run **expo/docs** app **[locally](https://github.com/expo/expo/tree/main/docs#running-locally)**.
- can run `expotools` locally, while in workspace
```

I think that being able to run Expo tools should also be listed as a perquisite.